src/brc20-bridge/src/canister.rs

```diff
@@ -73,16 +73,24 @@ impl Brc20Bridge {
             .map_err(GetAddressError::from)
     }
 
+    /// Retrieves all operations for the given ETH wallet address whose
+    /// id is greater than or equal to `min_included_id` if provided.
+    /// The operations are then paginated with the given `pagination` parameters,
+    /// starting from `offset` returning a max of `count` items
+    /// If `offset` is `None`, it starts from the beginning (i.e. the first entry is the min_included_id).
+    /// If `count` is `None`, it returns all operations.
     #[query]
     pub fn get_operations_list(
         &self,
         wallet_address: H160,
+        min_included_id: Option<OperationId>,
```

Comment:
When we created pagination people start complaining about arguments ordering, so we introduced `Pagination`, but now this could create the same situation.I don't know what you think about creating a `OperationListFilters` or something like that which wraps these two arguments.

Reply:
What was the complaint about exactly? I don't have any personal preference about arguments ordering in this endpoint, if there's one that better suits the user's needs, propose it and I'll update the signature. Btw, an `OperationListFilters` doesn't seem that much better to me.

Reply:
We had people not remembering whether to put offset or count first. But we can keep as-is as long as it doesn't create confusion